### {A Proposed Microbiota Stool Bus as a Mode of Infection for a Novel Posa-like Virus}
written by: [Derek Tea](https://github.com/DerekTea)

[Cover art Intestivirus](virus/img/p_intestivirus/cover_art_intestivirus.jfif)

### {Posasimilis Intestivirus}
Virus name: Posasimilis Intestivirus. I gave my virus this latin binomial name because it is 1) related to the unclassified family of posa-like viruses and 2) it, along with its relatives, are found in stool samples of its associated host. This relates to the story of my virus because I propose a mechanism whereby this virus, and presumably other posa-like viruses, infects the host through the gut microbiota (located in the intestinal tracts). 

## Abstract

RNA viruses have become the new craze in virology. Over the past few years, an RNA virus in particular, SARS-CoV-2, was the subject of widespread media coverage and scientific inquiry. After all, it was responsible for the spread and subsequent outbreak of a novel coronavirus that has put the world to its knees. Therefore, it would be advantageous to identify and characterize viruses well before they breach wider society. Despite their notoriety in nature, we are only scratching the surface of RNA virus discovery, as much looms in Earth's undiscovered virome. We used public sequencing databases to identify novel RNA viruses through candidate RNA-dependent RNA polymerase (RdRp) sequences. We uncovered a virus, tentatively named Posasimilis Intestivirus, that matched with rasavirus and other posa-like viruses. Previously, posa-like viruses were identified in the stool samples of their respective host organism. However, very little is known about their mechanism of action and biological relevance. Through sequence alignment tests of many viral "hit" databases, the Posasimilis Intestivirus genome was found in microbiome-inhabitant bacteria. These results suggest a potential mode of infection whereby Posasimilis Intestivirus infects microbiota populations in the gut and intestinal tracts. This supports the observations of posa-like viral species having only been sampled in fecal matter. Although experimental procedures are needed to validate such a proposal, it provides possible avenues to explore bacterial-viral relationships in the gut and its effects on host organisms.

## Results

The novel virus was discovered with the Serratus.io framework [Edgar, 2022]. In essence, the palmprint sequence of a detected RNA virus (which covered the RNA-dependent RNA polymerase structural motifs A, B, and C) was used to identify candidate viral RNA-dependent RNA polymerase (RdRp) sequences which correspond to a "node". The "node" identity was then cross-referenced to a database of assembled genomes to extract the full viral genome of interest. This protocol made way for the discovery of sequence runs (datasets) in which the virus is found. 

The index dataset has a run accession code of ERR3143221. Retrieved data from the EMBL-EBI database reveals that this particular sequence run originates from the paper "Discovery of novel astrovirus genotype species in small ruminants", published by Kauer et al in 2019. The study was performed in 2018 and employed Illumina sequencing on fecal matter from various ruminant species in Switzerland. This virus, tentatively named Posasimilis Intestivirus, derives from the Capra hircus (goat) genome. The paper sought to investigate the occurrence of neurotrophic astroviruses (single-stranded, positive-sense RNA viruses) in feces to characterize and analyze novel astroviruses and to examine potential interspecies transmission. This has implications in humans because astroviruses are known to cause diarrhea, and recent studies suggest an association between astroviral infection and encephalitis. However, the only neurotrophic astroviruses identified thus far were found in ruminants. The discovery of a new virus in this context would be valuable, as it may explain the transmission and/or mechanism by which these symptoms arise in humans. Naturally, it would be wise to determine whether virus ERR3143221 is indeed an astrovirus. 

The Posasimilis Intestivirus (ERR3143221) RdRp genomic sequence was run through Blastx (NCBI) and top hits suggest that this unknown virus is most similar to rasavirus sp, with a percent identity of ~30%. Since the virus of interest contains a candidate RdRp sequence, it is an RNA virus (Riboviria) [Venkataramen, 2018]. Rasaviruses are a species of RNA virus that is currently unclassified as very little is known of its genomic structure, mode of replication, and viral family. However, it branches off the Picornavirales order of viruses to form a cluster with other viruses - an unrecognized family known as "posa-like viruses" [Duraisamy, 2018]. Rasaviruses resemble posaviruses and other posa-like viruses, such as husavirus and fisavirus [Munnink, 2017]. The key difference is that rasaviruses are defined as rat-stool associated RNA viral genomes, whereas posaviruses (porcine stool), husaviruses (human stool), and fisaviruses (fish stool) are found in different host organisms [Munnink, 2017]. These viruses have been documented in areas around the globe, such as the USA [Shan, 2011], China [Zhang, 2014] [Han, 2020], and Vietnam [Munnink, 2017]. Although the genomic sequences of Picornavirales are highly divergent, the polyprotein sequence is usually conserved with a replication block of containing a helicase, protease, and RdRp [Han, 2020]. 

In general, posa-like viruses are not associated with disease and were first isolated in stool samples of healthy vertebrates. That is, posavirus was found in feces of healthy pigs, fisavirus in healthy carp, and husavirus in healthy humans [Munnink, 2017]. However, a study did find the presence of husavirus in children with acute gastroenteritis [Ramos, 2021]. 

To further confirm the genealogical identity of Posasimilis Intestivirus, its viral nucleotide sequence was converted into an amino acid equivalent with the OFRfinder tool (NCBI). The program searches for an 'ATG' start codon and counts the number of codons until it reaches a stop codon. The program identified 61 OFS, with ORF38 being the longest one and spanning 3006 amino acids (Figure 1). This amino acid sequence was ran back into Blastp (NCBI) and the resulting top hit produces a similar percent identity score with rasavirus sp. It hits a polyprotein found in rasavirus, as well as other polyproteins found in posa-like viruses further down the percent identity pecking order. Moreover, this protein product is closely related to other posa-like virus RdRps via the phylogenetic tree in NCBI (Figure 2). This provides further evidence that the virus of interest is a posa-like virus because amino acid similarity is often less than 40% within this group [Munnink, 2017]. 

Subsequent analysis into the other hit datasets may point to a viral mechanism of action by which this RNA virus ends up in its host organism. Whilst some datasets did not produce any candidate RdRp sequences, those datasets were all derived from sources related to fecal matter communities, with the exception of one (SRR3056448). Of the runs that did produce a candidate RdRp sequence, Blastx (NCBI) was used to isolate alignment top hits. 

In addition to dataset ERR3143221, the Posasimilis Intestivirus was also found in dataset ERR3143231, which belongs to the same study by Kauer et al. Notably, the source of this hit was Ovis aries (sheep) fecal matter rather than goat. Ensuing Blastx alignment tests match the candidate RdRp sequence with Caudovirales sp., Lachnospiraceae bacterium, and Bacteroidales bacterium as the top hits. These hits are peculiar as the percent identity score is two-fold greater compared to rasavirus sp. of the previous analysis. Furthermore, these hits are vastly different than posa-like viruses. For instance, Caudovirales sp. is species of virus within the Caudovirales order, a classification of double-stranded DNA bacteriophages [Tisza, 2021]. It could be that Posasimilis intestinum is an RNA bacteriophage. However, the known classes of RNA bacteriophages to date, Leviviridae (single-stranded RNA viruses) [Tars, 2020] and Cystoviridae [Mantynen, 2023], were not among the alignment match hits. Nonetheless, it remains a possibility that our virus of interest is in actuality an RNA bacteriophage that is evolutionarily related to double-stranded DNA bacteriophages. This proposal is supported by the Lachnospiraceae and Bacteroidales bacterium matches. Lachnospiraceae is abundant in the digestive tracts of mammals, humans included [Meehan, 2014], and Bacteroidales have been documented in the gastrointestinal tracts of homeothermic animals [Ormerod, 2016]. These bacteria are considered members of their host microbiome and could act as an avenue for which the virus of interest penetrates into the resulting fecal matter. The theory that posaviruses may infect a third-party rather than the host itself has previously been explored. Blast analysis of posavirus sequences has shown similarity to RNA sequences from a sample of the parasite Ascaris suum [Munnink, 2017]. 

The proposal for a bacterial-mediated entry into the host organism is reflected by the results from another dataset, SRR5830110 (accessible via the SRA database). The sample containing the Posasimilis Intestivirus hit derives from the fecal community of Tule Elk. Blastx analysis matches the virus with Caudovirales sp. and Candidatus Scalindua sp. as the top hits. The presence of Caudovirales may strengthen the notion that our virus of interest operates in a bacteriophage-like manner. Moreover, Candidatus Scalindua is a genus of marine-associated bacteria first discovered in a waster water treatment plant [Schmid, 2003]. This highlights the capacity for Posasimilis intestinum to infect bacteria, specifically bacteria that may be found in the water supply of wild-life. 

Related information pertaining to dataset runs can be found in the "MGY441 A3 Virus Seq Runs" csv file.

[MGY441 A3 Virus Seq Runs.csv](img/p_intestivirus/MGY441_A3_Virus_Seq_Runs.csv)
Table 1: A 'Top Hits' Analysis for Each P.Intestivirus Accession Code Hit. 

# Viral Genome 

[MGY441 A3 Viral Genome.png](img/p_intestivirus/MGY441_A3_Viral_Genome.png)
>Figure 1. A) Proposed Genomic Architecture of the Posasimilaris Intestivirus. Conserved sequences were identified with Blastp and aligned with the coding amino acid sequence of the virus. Distance representation between conserved regions are not to scale. The presence of the 5’NTR and 3’NTR is based on the conserved 5’NTR and 3’NTR domains found in Picornavirales [Jiang, 2014]. Size and length is not to scale.  B) Predicted RNA Secondary Structure of Posasimilis Intestiviridae. The prediction was performed with the RNAfold tool, utilizing minimal free energy predictions only. 

# Predicted 3D Structure
[Phyre2 3D Structure Motifs](img/p_intestivirus/Phyre2_3D_Structure_Motifs.png)
>Figure 2. A Predicted 3D Model of Posasimilaris Intestivirus. This model prediction was generated with the Phyre2 algorithm. 

# Posasimilaris Intestivirus RdRp Multiple Sequence Alignment 
[Multiple Sequence Alignment Jalview] (img/p_intestivirus/Multiple_Sequence_Alignment_Jalview.png)
>Figure 3. A Graphical Representation of the Posasimilaris Intestivirus RdRp Multiple Sequence Alignment. A multiple sequence alignment test was performed on the candidate Posasimilaris Intestivirus RdRp sequence using BLAST. The top hits, as defined by >80% query coverage and >40% amino acid identity, was visualized using the Jalview tool. 

# Predicted Phylogenetic Lineage
[Cobalt Phylogenetic Tree](img/p_intestivirus/Cobalt_Phylogenetic_Tree.png)
>Figure 4. The Predicted Posasimilaris Intestivirus Phylogenetic Lineage. This phylogenetic tree was generated on BLAST, following multiple sequence alignment of the Posasimilaris Intestivirus candidate RdRp sequence. 

## Discussion

Research into Posasimilis Intestivirus draws new insights into the characterization of this unclassified family. Very little research has been afforded to delineate the structure and viral properties of posa-like viruses. Specifically, previous findings into posa-like viruses had not described a means by which these viruses end up in the fecal matter of their respective host organism. Here, we propose a new trait, whereby a potential mode of transport into a host organism is through the infection of bacterial species in the microbiome. This finding is consistent with previous suggestions that posa-like viruses perpetuate via a third-party vector [Munnink, 2017]. As the current proposal is based solely on sequencing analyses, future experimental protocols are needed to establish a causative conclusion. If true, the mechanisms by which the virus enters the bacterium and replicates would be of interest. Moreover, whether Posasimilis Intestivirus, and subsequently other posa-like viruses, preferentially infect certain microorganisms would be useful. Should certain posa-like viruses preferentially infect microbiome populations, it may suggest an in vivo function and/or relationship on the overarching fitness and development of the host. 

## References

1. Edgar RC, Taylor B, Lin V, et al. 2022. Pentabase-scale sequence alignment catalyses viral discovery. Nature, 602:142-147. https://doi.org/10.1038/s41586-021-04332-2. 
2. Kauer RV, Koch MC, Hierweger MM, et al. 2019. Discovery of novel astrovirus genotype species in small ruminants. PeerJ, 7:e7338. doi: 10.7717/peerj.7338.
3. Venkataramen S, Prasad BVLS, Selvarajan R. 2018. RNA Dependent RNA Polymerases: Insights from Structure, Function and Evolution. Viruses, 10(2):76. doi: 10.3390/v10020076. 
4. Duraisamy R, Akiana J, Davoust B, et al. 2018. Detection of novel RNA viruses from free-living gorillas, Republic of the Congo: genetic diversity of picobirnaviruses. Virus Genes, 54(2): 256-271. doi: 10.1007/s11262-018-1543-6. 
5. Munnink BBO, Phan MVT, VIZIONS Consortinum, et al. 2017. Characterization of Posa and Posa-like virus genomes in fecal samples from humans, pigs, rats, and bats collected from a single location in Vietnam. Virus Evolution, 3(2):vex022, https://doi.org/10.1093/ve/vex022.  
6. Shan T, Li L, Simmonds T, et al. 2011. The fecal virome of pigs on a high-density farm. J Virol, 85(22):11697-708. doi: 10.1128/JVI.05217-11.
7. Zhang B, Tang C, Yue H. 2014. Viral metagenomics analysis demonstrates the diversity of viral flora in piglet diarrhoeic faeces in China. J Gen Virol, 95(Pt 7):1603-1611. doi: 10.1099/vir.0.063743-0. 
8. Han Z, Xiao J, Song Y. 2020. The Husavirus Posa-Like Viruses in China, and a New Group of Picornavirales. Viruses, 12(9): 995. doi: 10.3390/v12090995. 
9. Tisza MJ, Buck CB. 2021. A catalog of tens of thousands of viruses from human metagenomes reveals hidden associations with chronic diseases. Proc Natl Acad Sci U S A, 118(23): e2023202118. doi: 10.1073/pnas.2023202118. 
10. Tars K. 2020. ssRNA Phages: Life Cycle, Structure and Applications. Biocommunication of Phages, 30:261–92. doi: 10.1007/978-3-030-45885-0_13. 
11. Mantynen S, Salomaa MM, Poranen MM. 2023. Diversity and Current Classification of dsRNA Bacteriophages. Viruses, 15(11):2154.  https://doi.org/10.3390/v15112154. 
12. Meehan CJ, Beiko RG. A Phylogenomic View of Ecological Specialization in the Lachnospiraceae, a Family of Digestive Tract-Associated Bacteria, Genome Biology and Evolution, Volume 6, Issue 3, March 2014, Pages 703–713, https://doi.org/10.1093/gbe/evu050. 
13. Ormerod KL, Wood DLA, Lachner N. 2016. Genomic characterization of the uncultured Bacteroidales family S24-7 inhabiting the guts of homeothermic animals. Microbiome, 4(36). https://doi.org/10.1186/s40168-016-0181-2. 
14. Schmid M, Walsh K, Webb R. 2003. Candidatus “Scalindua brodae”, sp. nov., Candidatus “Scalindua wagneri”, sp. nov., Two New Species of Anaerobic Ammonium Oxidizing Bacteria. Systematic and Applied Microbiology, Volume 26, Issue 4, Pages 529-538, https://doi.org/10.1078/072320203770865837. 
15. Ramos ESF, Rosa UA, Ribeiro GO. 2021. Multiple clades of Husavirus in South America revealed by next generation sequencing. PLos One.  https://doi.org/10.1371/journal.pone.0248486
16. Aoki H, Sunaga F, Ochiai H, et al. 2019. Phylogenetic analysis of novel posaviruses detected in feces of Japanese pigs with posaviruses and posa-like viruses of vertebrates and invertebrates. Arch Virol, 164:2147–2151. https://doi.org/10.1007/s00705-019-04289-8. 
17. Yinda CK, Zell R, Deboutte W. 2017. Highly diverse population of Picornaviridae and other members of the Picornavirales, in Cameroonian fruit bats. BMC Genomics, 18:249. doi 10.1186/s12864-017-3632-7. 
18. Jiang P, Liu Y, Ma HC, et al. 2014. Picornavirus Morphogenesis. Microbiol Mol Biol Rev, 78(3): 418–437. doi: 10.1128/MMBR.00012-14. 
19. Witwer C, Rauscher S, Hofacker IL, et al. 2001. Conserved RNA secondary structures in Picornaviridae genomes. Nucleic Acids Res., 29(24): 5079–5089.
doi: 10.1093/nar/29.24.5079. 

# Viral Short Story

```
There once was a virus that no one knew. It wasn't unpopular, just too shy to say hi to you! It traveled far and wide to find a home, yet no one was accepting and so it remained alone. People and animals were scared alike, as its name - **Rasavirus**, was hard to like. It was a social outcast with **no recognized family**, only a bunch of look-a-likes...oh what a misery! **Posavirus, husavirus, fisavirus...so many species!** Yet they were able to find homes in the unlikeliest of place...**feces**! Each of them finding their way into animals, a peculiar tactic that may seem irrational. But Rasa saw it all and wondered to itself, could it find a home too? Even in a clump of doo-doo?! But that was no fun, it would just smell so bad! And so it thought and sat until it turned sad. There was no other way around it...until suddenly it had a great idea. It would disguise itself in food and make its way into animals. Then help their digestive tract, gut virome, and microbiota! So Rasa did just that. And sooner than not, it was in **sheep, elk, and yak**! None of these animals had any idea, and they remained **healthy**...no diarrhea! And that was the story of Rasa, so what's there to learn? That not all viruses are bad, and some just need a home... even if its in your turd!
```
